Understanding OCSF.md

```diff
@@ -229,27 +222,35 @@ There are three important attributes in the Observable object: `name`, `value`,
 
 For complex (object type) attributes, Observable.`name` is the pointer or reference to the attribute, but as an object has more than one value, Observable.`value` is not populated.
 
-
-```json
-"observables": [
-{ "name": "actor.process.name",
-"type": "Process Name",
-"type_id": 9,
-"value": "Notepad.exe"
-},
-{ "Name": "tls.ja3_hash",
-"Type": "Fingerprint",
-"Type_id": "30"
-},
-{ "name": "file.name",
-"type": "File Name",
-"type_id": 7,
-"value": "Notepad.exe"
-}]
+```jsonc
+{
+  // ... (OCSF event fields)
+  "metadata": {
```

Comment:
Any reason why we are nesting it in `metadata`? `observables` is a base event attribute.

Reply:
Good catch. Fixed.